frontend/src/main/scala/bloop/exec/Forker.scala

```diff
@@ -195,7 +195,7 @@ object Forker {
           gobbleInput.cancel()
           try process.closeStdin(true)
           finally {
-            process.destroy(true)
+            process.destroy(false)
             process.waitFor(200, _root_.java.util.concurrent.TimeUnit.MILLISECONDS)
```

Comment:
I also propose we increase the timeout to 1s as 200 ms is rather short.

Reply:
1s is too much, 400ms is a good time. Also note that the JVM shutdown hooks give no guarantee to the programmer about how much time they are executed or if they are executed at all, so I believe 400ms is a good compromise.

Reply:
Okay, I will change it to 400ms.